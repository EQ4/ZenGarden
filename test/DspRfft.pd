#N canvas 632 261 362 216 10;
#X obj 51 39 osc~ 440;
#X obj 51 82 rfft~;
#X obj 51 130 print~ real;
#X obj 163 131 print~ imaginary;
#X obj 131 37 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 131 80 t b b;
#X obj 154 36 loadbang;
#X connect 0 0 1 0;
#X connect 1 0 2 0;
#X connect 1 1 3 0;
#X connect 4 0 5 0;
#X connect 5 0 2 0;
#X connect 5 1 3 0;
#X connect 6 0 5 0;
