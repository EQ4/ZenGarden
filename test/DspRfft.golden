real:
-11.3715 9.22387 2.21074 1.40507 1.14521 1.0285 0.966014 0.928656 0.904543 0.88808 0.876346 0.867694 0.861137 0.856055 0.852042 0.848825 0.846211 0.844065 0.842287 0.840804 0.839559 0.838512 0.837629 0.836886 0.836259 0.835736 0.835304 0.834953 0.834675 0.834464 0.834316 0.834229 0.8342 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
imaginary:
0 -28.2682 -9.29866 -5.80785 -4.24303 -3.33798 -2.74243 -2.31799 -1.9984 -1.74781 -1.54508 -1.37689 -1.23446 -1.1117 -1.00432 -0.90915 -0.82383 -0.746544 -0.675883 -0.610724 -0.550166 -0.493474 -0.440037 -0.389345 -0.340963 -0.294518 -0.249683 -0.206168 -0.163715 -0.122087 -0.0810609 -0.040432 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0